README.md

```diff
@@ -2,3 +2,27 @@
 
 [![Join the chat at https://gitter.im/hitraffic/mobile](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/hitraffic/mobile?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 A hybrid mobile app powered by the Ionic Framework.  Utilizes the hitraffic API to provide traffic incident reports.
+
+---
+
+Installation
+-----------------------------
+
+1. Fork and clone this repo
+2. Run `npm install`
+3. Run `bower install`
+4. Run `ionic lib update` and enter `yes` when prompted
```

Comment:
I don't think people should be updating the lib files. It looks like `www/lib` should be committed.
After running `ionic start` the default `.gitignore` only contains:

```
node_modules/
platforms/
plugins/
```